Modules/Sources/DesignSystem/Components/DSButton.swift

```diff
@@ -4,19 +4,22 @@ public struct DSButton: View {
     @Environment(\.isEnabled) private var isEnabled
     @Environment(\.colorScheme) private var colorScheme
     private let title: String
+    private let iconName: IconName?
     private let style: DSButtonStyle
     @Binding private var isLoading: Bool
     private let action: (() -> Void)
 
     public init(
         title: String,
+        iconName: IconName? = nil,
```

Comment:
It's be great to allow using SF symbols.

Reply:
@kean, what do you think about adding a plus sign icon to `IconName`, and then updating the code responsible for loading the image to either load it from `xcassets` or as an SF symbol? Having all icons in one place would help in knowing which icons are used in the app.

Reply:
I agree with Salim. This way we can have an exhausting list whereas with direct strings, we can't know without digging into the code, which icons are being used in the app. To address the issue Salim mentioned, I'll try to come up with a solution to make the difference between a custom image and sfsymbol explicit.